### ISSUE TITLE
deprecation of delayed and static scaling from torchao.float8

### ISSUE THREAD

**Issue by vkuzo** [pytorch/ao#1680]
# Summary

We plan to deprecate delayed and static scaling from `torchao.float8` training codebase due to lack of real world use cases for delayed/static scaling (dynamic scaling is required for higher accuracy).

# Deprecation timeline

* v0.9.0: display deprecation warning
* v0.10.0: deprecate

# Alternatives

Use dynamic scaling

**Comment by vkuzo** [comment]
this is done
